Samples/Entities/Entities.csproj

```diff
@@ -1,12 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net462</TargetFrameworks>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\Source\Svg.csproj" />
-  </ItemGroup>
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFrameworks>netcoreapp3.1;net462</TargetFrameworks>
+	</PropertyGroup>
+
+	<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+		<NoWarn>$(NoWarn);CS0618;NETSDK1138</NoWarn>
+	</PropertyGroup>
+	<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+		<NoWarn>$(NoWarn);CS1591;CS0618;NETSDK1138</NoWarn>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<ProjectReference Include="..\..\Source\Svg.csproj" />
+	</ItemGroup>

```

Comment:
You have changed spaces to tabs in the project files. Is this intentional?

Reply:
Yes, using the default 4 spaces tab makes it easier to edit the files. The default indentation size in VS is 4 spaces unless changed by a user.
The project file indentation was 2 spaces.